app/controllers/questions_controller.rb

```diff
@@ -14,6 +14,8 @@ def show
 
   # GET /questions/new
   def new
+    questionFile = File.read('config/questions.json')
+    @questionsHash = JSON.parse(questionFile)
```

Comment:
'questionHash' makes it seem like the hash value of a question. Let's rename this to something that accurately represents the contents.

Reply:
This is indeed a hash (dictionary . So this name is the mos apt. Take a look at json.